src/CRM/CivixBundle/Resources/views/Code/module.civix.php.php

```diff
@@ -154,7 +154,7 @@ function _<?php echo $mainFile ?>_civix_civicrm_postInstall() {
  *
  * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_uninstall
  */
-function _<?php echo $mainFile ?>_civix_civicrm_uninstall() {
+function _<?php echo $mainFile ?>_civix_civicrm_uninstall(): void {
```

Comment:
Long history of no return 

![image](https://user-images.githubusercontent.com/336308/196287186-b84e34ed-d347-46a1-8889-b432efde74c5.png)